cogs/tracking.py

```diff
@@ -1282,11 +1280,18 @@ def __init__(self, bot: NabBot):
                     continue
                 await conn.execute("INSERT INTO character_death_killer(death_id, name, player) VALUES($1, $2, $3)",
                                    death_id, death.killer, death.by_player)
-                if time.time() - death.time.timestamp() >= (30 * 60):
-                    log.info(f"Death detected, too old to announce: {char.name}({death.level}) | {death.killer}")
+                log_msg = f"Death detected: {char.name}({death.level}) | {death.killer}"
+                if self.is_old_death(death):
+                    log_msg += ", but it is too old to announce."
                 else:
+                    log.info(log_msg)
```

Comment:
It seems like no log message will be printed if the death is too old, even thought you're appending something to `log_msg`.

Reply:
My bad, fixed now. Should only log when it is old and log + announce when it is ok.